core/src/04-wealth-equipment/01-chapter-four.md

```diff
@@ -229,7 +229,7 @@ Typically a weapon belongs to one category, however some weapons, such as a dagg
 
 Every weapon must have either the *Forceful* or *Precise* property and some weapons have both. Choose one or both.
 
-Next, you will may select a number of properties that have significant game impact, just like *Forceful* and *Precise*, these are listed under properties. The difference is that these other properties each have the effect of raising the Wealth Level of the weapon.
+Next, you will may select a number of properties that have significant game impact, just like *Forceful* and *Precise*, these are listed under properties. The difference is that these other properties each have the effect of raising the Wealth Level of the weapon. The Wealth Level of all weapons begins at 1.
```

Comment:
Pretty sure this should say "All weapons have a minimum Wealth Level of 1." 

Otherwise, if starting point is 1, then `Rocket Launcher, Mech Warhead` would be:

  1
+4 Deadly 2
+3 Burst 3
-1 Delayed Ready
-1 Heavy
-1 Consumable
_________
5 (total)